docs/core/rid-catalog.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: .NET Core Runtime IDentifier (RID) catalog
+title: .NET Core Runtime Identifier (RID) catalog
```

Comment:
Should we just change it in the rest of the article too? I think it was there to show how RID is broken out... which seems a bit pointless.

Reply:
Yes, I didn't notice that it was done that way in another place.  Fixed it.